Makefile

```diff
@@ -91,7 +96,12 @@ CONTAINER_RUN ?= $(CONTAINER_CMD) --security-opt label=disable -v $(CURDIR):$(CO
 GIT_COMMIT := $(shell git rev-parse HEAD 2> /dev/null || true)
 
 EXTRA_LDFLAGS ?=
-SKOPEO_LDFLAGS := -ldflags '-X main.gitCommit=${GIT_COMMIT} $(EXTRA_LDFLAGS)'
+SKOPEO_LDFLAGS := -ldflags '-X main.gitCommit=${GIT_COMMIT} \
+				  -X github.com/containers/image/v5/docker.systemRegistriesDirPath=${REGISTRIESDDIR} \
+				  -X github.com/containers/image/v5/signature.systemDefaultPolicyPath=${CONTAINERSCONFDIR}/policy.json \
+				  -X github.com/containers/image/v5/pkg/sysregistriesv2.systemRegistriesConfPath=${CONTAINERSCONFDIR}/registries.conf \
+				  -X github.com/containers/image/v5/internal/tmpdir.unixTempDirForBigFiles=/var/tmp \
```

Comment:
This overrides the hard-coded path with the same hard-coded path, AFAICS; why?

Reply:
@mtrmac so 

> This doesn’t seem to resolve the concerns in [#1298 (comment)](https://github.com/containers/skopeo/pull/1298#issuecomment-849529062) ; just silently switching a default on an upgrade, and having defaults different from Buildah/Podman (ideally from the c/image builtins), is not desirable.
> 
> Making that change exclusive to macOS and FreeBSD makes it less problematic for Linux, but still problematic for the two, I think.

@mtrmac so would you prefer the same change be done across the board? If macos and freebsd prefer it in /usr/local, then I imagine this change has to be done at some point, yes?

The extra ldflag lines were a copy-paste from the older PR. I could make those conditional for homebrew / FreeBSD if they prefer to have them. @glensc @mateuszkwiatkowski wdyt?

Reply:
> If macos and freebsd prefer it in /usr/local, then I imagine this change has to be done at some point, yes?

No?

Users who use Brew or other build systems that already override the paths can continue to use those build systems. Users who use `make install` would be broken on upgrade if we just changed the default with no compatibility mechanism; so we shouldn’t just change the default.

My view of #1298 is that adding reliable variables (like `CONTAINERSCONFDIR` or whatever), which would help Brew reliably work, is obviously useful and desirable; changing what the variable point to, by default, is not.

Reply:
Alternatively, we could work to actually provide a compatibility mechanism (i.e. support at least two paths in c/image, and configure them to… something in there and possibly here). I don’t see that that work and complexity helps any known existing user — though it would help users who cross-grade between packaging systems like Brew and manual `make install`.